apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/preadj/common/ClaimDao.java

```diff
@@ -68,6 +78,75 @@ public Object getEntityById(ResourceTypeV2<?> type, String id) {
     return claimEntity;
   }
 
+  public <T> List<T> findAllByMbiHash(
+      Class<T> entityClass, String mbiHash, DateRangeParam lastUpdated) {
+    return findAllByAttribute(entityClass, "mbiHash", mbiHash, lastUpdated);
+  }
+
+  public <T> List<T> findAllByMbi(Class<T> entityClass, String mbi, DateRangeParam lastUpdated) {
+    return findAllByAttribute(entityClass, "mbi", mbi, lastUpdated);
```

Comment:
Will the attribute also be named "mbi" in the MCSClaim DAO?

Reply:
At the time this was created, I had no idea, so it was anticipated there may be some rework needed when the MCS implementation came around, which is now looking likely.